app/src/pages/inside/projectSettingsPageContainer/content/notifications/ruleGroup/ruleGroup.jsx

```diff
@@ -0,0 +1,335 @@
+          onSave: confirmEdit,
+          notification,
+          notifications,
+        },
+      }),
+    );
+  };
+
+  const onDelete = (notification) => {
+    trackEvent(PROJECT_SETTINGS_NOTIFICATIONS_EVENTS.CLICK_ICON_DELETE_NOTIFICATIONS);
+
+    dispatch(
+      showModalAction({
+        id: 'deleteNotificationModal',
+        data: {
+          onSave: () => confirmDelete(notification.id),
+        },
+      }),
+    );
+  };
+  const onCopy = (notification) => {
+    trackEvent(PROJECT_SETTINGS_NOTIFICATIONS_EVENTS.CLICK_ICON_DUPLICATE_NOTIFICATIONS);
+
+    const { id, ...newNotification } = notification;
+    dispatch(
+      showModalAction({
+        id: 'addEditNotificationModal',
+        data: {
+          actionType: MODAL_ACTION_TYPE_COPY,
+          onSave: (withoutAttributes) => confirmAdd(withoutAttributes),
+          notification: {
+            ...newNotification,
+            ruleName: notification.ruleName + COPY_POSTFIX,
+          },
+          notifications,
+        },
+      }),
+    );
+  };
+
+  const actions = [
+    {
+      icon: CopyIcon,
+      handler: onCopy,
+      dataAutomationId: 'duplicateNotificationRuleIcon',
+    },
+    {
+      icon: PencilIcon,
+      handler: onEdit,
+      dataAutomationId: 'editNotificationRuleIcon',
+    },
+    {
+      icon: BinIcon,
+      handler: onDelete,
+      dataAutomationId: 'deleteNotificationRuleIcon',
+    },
+  ];
+
+  return (
+    <div className={cx('rule-section')}>
+      <Layout description={''} className={cx('rule-section-layout')}>
+        <div className={cx('rule-section-header')}>
+          <FieldElement
+            className={cx('fieldElement')}
+            withoutProvider
+            description={formatMessage(messages.typeDescription, { type: pluginName })}
+          >
+            <div className={cx('toggle')}>
+              <Toggle
+                disabled={isReadOnly}
+                value={isPluginNotificationsEnabled}
+                onChange={(e) => togglePlugin(e.target.checked)}
+                dataAutomationId="enabledToggle"
+              >
+                <span className={cx('name-wrapper')}>
+                  <i className={cx('capitalized')}>{pluginName}</i>
+                </span>
+              </Toggle>
+            </div>
+          </FieldElement>
+
+          {!isAvailable() ? (
+            <div className={cx('disabled-plugin')}>
+              <p>
+                <span className={cx('capitalized')}>{pluginName}</span>{' '}
+                {formatMessage(messages.disabledPlugin, { pluginName })}
+              </p>
+              <RuleItemDisabledTooltip
+                className={cx('info-tooltip')}
+                tooltipContent={formatMessage(messages.disabledContactInfo)}
+              >
+                <i className={cx('icon', 'about-icon')}>{Parser(AboutIcon)}</i>
+              </RuleItemDisabledTooltip>
+            </div>
+          ) : (
+            pluginName === 'email' &&
```

Comment:
I suggest to use a `EMAIL` constant here from `src/common/constants/pluginNames.js`.